list.txt

```diff
@@ -78,21 +86,22 @@ holistic-partners.com
 iconstafflabs.com
 idctechno.com
 idctechnologies.com
+iic.com
 imipeople.com
 inabia.com
 infinite-usa.com
 infojiniconsulting.com
 infoorigin.com
 infosoft-inc.com
 infosys.com
+insightglobal.com
```

Comment:
These guys... They hire models that LARP as recruiters LOL.

Reply:
Just noticed they spammed me on LinkedIn too, but I must have deleted their spammy message they directly sent me.